tests/vm_watch_test.go

```diff
@@ -21,338 +24,337 @@ const (
 	relevantk8sVer = "1.16.2"
 
 	// Define a timeout for read opeartions in order to prevent test hanging
-	readTimeout     = 1 * time.Minute
-	processWaitTime = 2 * time.Minute
+	readTimeout       = 30 * time.Second
+	vmCreationTimeout = 1 * time.Minute
+	vmAgeRegex        = "^[0-9]+[sm]$"
+	ipRegex           = "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$"
```

Comment:
This only works with IPv4.
Maybe `net.ParseIP()` could be used instead? (https://golang.org/pkg/net/#ParseIP)

Reply:
Do you know how would I go using `net.ParseIP()` in this assertion:
```
Expect(vmiStatus).To(ConsistOf(vmi.Name, MatchRegexp(vmAgeRegex), string(v12.Scheduled), MatchRegexp(ipRegex), vmi.Status.NodeName),
			"VMI should be in the Scheduled phase")
```
Maybe I should just use a regex that matches both IPv4 and IPv6?

Reply:
Right, to use `ParseIP()` you'd still need to parse the string into fields... That might still be worth it, at least for tests which check IPs.
With that I would ParseIP() on fields[3] to `ip`, Expect() it to be non-nil, and in the ConsistOf, use ip.String() (or just fields[3]) to match the IP.
Expending that regex with a big "or" block for ipv6 would make the string pretty huge!

Reply:
I just now saw your reply, sorry about that!
I like your idea, will work on it right now